### PR TITLE
Create animation presets for BottomSheetDialog

### PR DESCRIPTION
**Pull request by everdimension**

### CODE REVIEW

File: src/ui/ui-kit/ModalDialogs/BottomSheetDialog/styles.module.css [hunk 1]
```diff
@@ -9,11 +9,25 @@
...
+  }
+}
+
 .slideUp {
   will-change: transform, opacity;
   animation: slideUp 150ms;
 }
 
+.slideUpNoFadeIn {
+  will-change: transform, opacity;
```
Comment (zerts) [inline src/ui/ui-kit/ModalDialogs/BottomSheetDialog/styles.module.css:27]: Should we mention opacity here?

Comment (everdimension) [reply src/ui/ui-kit/ModalDialogs/BottomSheetDialog/styles.module.css:27]: ah, good catch

File: src/ui/ui-kit/ModalDialogs/BottomSheetDialog/BottomSheetDialog.tsx [hunk 2]
```diff
@@ -4,23 +4,35 @@ import type { BaseDialogProps } from '../BaseDialog';
 import { BaseDialog } from '../BaseDialog';
 import * as s from './styles.module.css';
 
+export enum AnimationPreset {
```
Comment (zerts) [inline src/ui/ui-kit/ModalDialogs/BottomSheetDialog/BottomSheetDialog.tsx:7]: Do we have any examples of different styles in the extension right now?

Comment (everdimension) [reply src/ui/ui-kit/ModalDialogs/BottomSheetDialog/BottomSheetDialog.tsx:7]: We're about to ;)

**Pull request merged**